cmd/logger.go

```diff
@@ -160,10 +196,20 @@ func logIf(level logrus.Level, source string, err error, msg string, data ...int
 		"cause":  err.Error(),
 	}
 
```

Comment:
fields is not used during `json` mode move this struct below after the `if log.json` conditional.